spec/models/order_spec.rb

```diff
@@ -84,5 +84,19 @@
       expect(order.current_status).to eq("Packaged")
 
     end
+
+    it 'items_by_merchant' do
+      expect(@order_2.items_by_merchant(@meg.id)).to eq(2)
+      expect(@order_1.items_by_merchant(@meg.id)).to eq(2)
```

Comment:
Items by merchant is the sum of the quantity of all the items for a single merchant in an order